splunk-adx-alert-addon/adx_connector.py

```diff
@@ -0,0 +1,155 @@
+import io
+import json
+import os
```

Comment:
A generic README.txt would be great too, articulating functionality as well as build steps etc.

Reply:
Added a README describing the steps. Since the build is governed from Splunk Appbase didn't mention on that.